django/forms/boundfield.py

```diff
@@ -231,11 +220,27 @@ def id_for_label(self):
         id_ = widget.attrs.get('id') or self.auto_id
         return widget.id_for_label(id_)
 
+    @property
```

Comment:
I didn't see the first version of the PR which might have addressed this but it looks like `self._initial_value` is emulating `@cached_property`. To be improved on master...